recipes/timescoring/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "timescoring" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/timescoring-{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Reply:
Thanks for the review. I modified the recipe as per your suggestion